lib/yaml-parser/index.ts

```diff
@@ -3,6 +3,9 @@ import * as YAML from 'yaml';
 export function parseFileContent(fileContent: string): any[] {
   // the YAML library can parse both YAML and JSON content, as well as content with singe/multiple YAMLs
   // by using this library we don't have to disambiguate between these different contents ourselves
+  if (fileContent.includes('\\/')) {
```

Comment:
As this is a pretty unusual thing to see in the code could you add a comment with a link to the original ticket that describes why we need to implement this behaviour so it's not accidentally removed in the future.